Library/Homebrew/utils/bottles.rb

```diff
@@ -116,7 +116,11 @@ def load_tab(formula)
           tab_json = bottle_hash[formula.full_name]["bottle"]["tags"][tag]["tab"].to_json
           Tab.from_file_content(tab_json, tabfile)
         else
-          Tab.for_keg(keg)
+          tab = Tab.for_keg(keg)
+          f_runtime_deps = formula.runtime_dependencies(read_from_tab: false)
```

Comment:
Should/could this verify if any runtime dependencies are already present in the tab first?

Reply:
Ahh this is a great suggestion, I just realized it'll only be `empty` if it doesn't come from a file / path doesn't exist.

https://github.com/Homebrew/brew/blob/11e4b66dcfacd8740b56df79cae1673b22143a7b/Library/Homebrew/tab.rb#L119-L130

I'll make this change